systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlST.java

```diff
@@ -142,6 +146,52 @@ void testAutoCreationOfCruiseControlTopicsWithResources(ExtensionContext extensi
         assertThat(partitionMetricsTopic.getReplicas(), is(2));
     }
 
+    @IsolatedTest
+    @KRaftNotSupported("Topic Operator is not supported by KRaft mode and is used in this test class")
```

Comment:
Why does this use Topic Operator? And maybe a bit more general, why does this need a separate System test? I would think that:
* Unit tests should be sufficient to cover this
* If we want a system test, it can be probably added to one of the existing tests?